int32 device_id
int32 task_id
int32 map_id
string task_name
int32 task_code
float32 task_progress
int32 task_status
string message

